Updates German localization

brandl-muc opened the pull request:
Adds missing German translations and updates some of the existing translations

da2x commented:
Thanks, @brandl-muc! I’ve update master based on your comments and corrected some odd examples. Could you please rebase your translations?

Oh, and slow-time is a digital approximation of an analog 24-hour linear clock that don’t give you more precision than ¼ increments in an hour. The idea is to make time progress slower, stop thinking about precise time and rather focus on getting your work done instead.

Some example analog watches (can’t implement these given the constraints of the Shell and the limited available screen space):
https://de.slow-watches.com/de/the-store/slow-round-02/
https://www.botta-design.de/en/uhrenmodelle/uno-24/uno24-neo-black-edition-quarz/a-162/
http://svalbard.watch/pages/Svalbard_Solfestuka_CA12.html

brandl-muc commented:
All right, rebased on the changed header. I squashed my old changes into the new commit which updated the PO file with the POT file. I also changed the comments where appropriate.

I actually know the Slow watches, but more as 24 h clock face watches with phase shift. It didn't occur to me that you were referring to them.

da2x commented:
Thanks!
